scss/main.scss

```diff
@@ -1,3 +1,4 @@
 @import 'html5-bp_main';
-@import 'unhcr';
-@import 'user_location_control';
\ No newline at end of file
+@import 'user_location_control';
+@import url(http://fonts.googleapis.com/css?family=Open+Sans:400,300,700);
+@import 'unhcr_new';
```

Comment:
I've removed things in _unhcr_new that is required to run the existing
site. I switch back to it occasionally for reference, hence keeping it
until I don't need it anymore.

On Tue, Apr 14, 2015, 10:46 PM Luke Kysow <notifications@github.com> wrote:

> In scss/main.scss
> <https://github.com/PeaceGeeksSociety/services-advisor/pull/2#discussion_r28394769>
> :
>
> > \ No newline at end of file
> > +@import 'user_location_control';
> > +@import url(http://fonts.googleapis.com/css?family=Open+Sans:400,300,700);
> > +@import 'unhcr_new';
>
> I think you can just replace _unhcr.scss with your new one unless there's
> a reason to keep the old one?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PeaceGeeksSociety/services-advisor/pull/2/files#r28394769>
> .
>